gcd/src/test/java/org/spine3/server/storage/datastore/DsAggregateStorageShould.java

```diff
@@ -65,14 +65,16 @@ public static void tearDownClass() {
         datastoreFactory.tearDown();
     }
 
-    @SuppressWarnings("ConstantConditions") // passing null because this parameter isn't used in this implementation
+    @SuppressWarnings("ConstantConditions")
+    // passing null because this parameter isn't used in this implementation
```

Comment:
Please have a tabulation (4 spaces) at the start of the line. This is required to make it obvious that the comment belongs to this specific suppression.